src/packages/core/section/section-views/section-views.element.ts

```diff
@@ -141,16 +147,17 @@ export class UmbSectionViewsElement extends UmbLitElement {
 		return this._dashboards.length > 0
 			? html`
 					<uui-tab-group slot="header" id="dashboards">
-						${this._dashboards.map(
-							(dashboard) => html`
+						${this._dashboards.map((dashboard) => {
+							const dashboardName = dashboard.meta.label ?? dashboard.name;
+							const dashboardPath =
```

Comment:
Looks like we are constructing this path more than once. This could be prone to errors. Have you considered making a common function to construct the dashboardPath?

Reply:
Yeah, for sure. I have made two methods for it.

We might come up with some smarter methods, enabling similar for multiple other implementations.

As I think manifests, only should be defining pathnames, if the developer likes to be specific. otherwise, we should just generate it from data. So I think we will see more of this in the future. :-)